### ISSUE TITLE
Gold standard handling removal of [] for references

### ISSUE THREAD

**Issue by jameshowison** [jameshowison/softcite#6]
How should I hadnle the situations where pdf_to_text has removed the square braces from references?

e.g.: 2004-46-NATURE-C01-mention:

<mention id="bioj:a2004-46-NATURE-C01-mention" original_name="PONDR">The molecule is rich in proline residues (13%) and analysis of its amino acid sequence with PONDR21 indicates that in the absence of other viral components at least the N-terminal half of the subunit would be disordered.</mention>

Should I do anything to indicate that here the 21 was actually a reference?  I don't know that stemming is going to work because some software does end in numbers.  I wonder whether some tweak to the pdf_to_txt code might help us here?

**Comment by jameshowison** [comment]
Hmmm, but if I add those manually to the labeled ones, they won't be in the unlabelled sentences...that doesn't seem right to me.  Looking at 2004-46-NATURE.pdf the PONDER21 thing comes from PONDER superscript 21.  Not sure why pdf_to_text is ignoring the superscript.

Anyway, Can you think about whether correcting these but not correcting things in other sentences would be a problem?

**Comment by yg4886** [comment]
Yes, I think you are right.  The test dataset is generated by pdf_to_text too. So we need to use the manually labeled ones to predict the pdf_to_text generated sentences(test dataset) that still contain uncertainty. But, I am not sure, maybe the manual work would increase the accuracy. 

And yes, the pdf_to_text is annoying. But I think there should always be some noises in the training dataset in practice. Sometime it is difficult to decrease the noise, i.e. in our project, the cost is expensive. (there may be cheap ways to do that. we can use AMT to transfer all the pdfs to txts). We’ll always train the model with data containing noise. And then try anything we can do to improve its performance.  

And, also, Byron, do you have any comments?

-Yan
